src/main/webapp/app/entities/feedback.model.ts

```diff
@@ -25,6 +25,27 @@ export interface DropInfo {
     instruction: GradingInstruction;
 }
 
+/**
+ * Possible tutor feedback states upon validation from the server.
+ */
+export enum FeedbackCorrectionErrorType {
```

Comment:
This is exactly would I would suggest for the server enum type as well, is something wrong with having the same name? I find `FeedbackCorrectionErrorType` suitable - provided that you do not include a case for 'CORRECT' as I suggested. If you do add the 'CORRECT' / 'NO_ERROR' case at the server, you should also do this on the client.
`*ngIf="feedback.correctionStatus != undefined"` would then be `*ngIf="feedback.correctionStatus != CORRECT`

Reply:
Understood.